packages/build-utils/test/fixtures/24-pnpm-pnp-symlink/vercel.json

```diff
@@ -0,0 +1,10 @@
+{
+  "version": 2,
+  "builds": [{ "src": "package.json", "use": "@vercel/static-build" }],
+  "probes": [
+    {
+      "path": "/",
+      "logMustContain": "foobar"
```

Comment:
You also need to test the api endpoints to see if the packages were installed properly